weaviate/collections/batch/grpc_batch_objects.py

```diff
@@ -152,6 +153,8 @@ async def __send_batch(
                 objects[result.index] = result.error
             return objects
         except AioRpcError as e:
+            if e.code().name == "PERMISSION_DENIED":
```

Comment:
can we do a constant instead?

Reply:
As in, define it in `shared.py` and import it everywhere? Can do, `e.code().name` here is a string literal in the typing system so this line is covered by `mypy`